client/src/components/composite/Booking/BookingInfoComponent/BookingInfoComponent.tsx

```diff
@@ -0,0 +1,52 @@
+interface IBookingInfoProps {
+  price1: string
+  price2: string
+  price3: string
+}
+
+type props = IBookingInfoProps
+
+const Divider = () => (
+  <span className="bg-dark-blue-100 m-8 h-[1px] w-[283px]" />
+)
+
+const BookingInfoComponent = ({ price1, price2, price3 }: props) => {
+  return (
+    <div className="border-gray-3 flex h-[474px] max-w-[347px] flex-col items-center rounded border">
```

Comment:
this should be full width with content being centered on smaller screen so its responsive. height should be set to fit content with appropriate paddings